js/src/jupyter-leaflet.js

```diff
@@ -705,6 +706,51 @@ var LeafletLayersControlView = LeafletControlView.extend({
     }
 });
 
+var LeafletMeasureControlView = LeafletControlView.extend({
+    initialize: function (parameters) {
+        LeafletMeasureControlView.__super__.initialize.apply(this, arguments);
+        this.map_view = this.options.map_view;
+    },
+
+    render: function () {
+        this.create_obj();
+        this.model_events();
+    },
+
+    create_obj: function () {
+        this.obj = L.control.measure(this.get_options());
+        this.obj.addTo(this.map_view.obj);
+        this.default_units = L.extend({}, this.obj.options.units);
+    },
+
+    get_options: function () {
+        var o = this.model.get('options');
+        var options = {};
+        var key;
+        for (var i=0; i<o.length; i++) {
+            key = o[i];
+            // Convert from foo_bar to fooBar that Leaflet.js uses
+            options[camel_case(key)] = this.model.get(key);
+        }
```

Comment:
maybe call super.get_options instead of duplicating the implementation.

Reply:
There is no such method in ControlView, but I will add it. Maybe it will factorize some code 😊